src/Symfony/Component/Intl/Resources/emoji/README.md

```diff
@@ -0,0 +1,19 @@
+# Emoji Transliterator Builder
+
+This folder contains the tool to build all transliterator rules.
+
+## Requirements
+
+* composer
+* PHP
+
+## Update the rules
+
+To update the rules, you need to update the version of `iamcal/emoji-data` and
```

Comment:
There is no `iamcal/emoji-data` in it.

Reply:
Good catch. It was a relicat from a previous version of a next PR (wait, what? 😂 ). Now I download directly what I need!